src/main/drivers/pwm_output.h

```diff
@@ -29,6 +29,26 @@
 #endif
 
 typedef enum {
+    DSHOT_CMD_MOTOR_STOP = 0,
+    DSHOT_CMD_BEEP1,
+    DSHOT_CMD_BEEP2,
+    DSHOT_CMD_BEEP3,
+    DSHOT_CMD_BEEP4,
+    DSHOT_CMD_BEEP5,
+    DSHOT_CMD_ESC_INFO, //NOT USED
+    DSHOT_CMD_SPIN_ONE_WAY, //NOT USED
+    DSHOT_CMD_SPIN_OTHER_WAY, //NOT USED
+    DSHOT_CMD_3D_MODE_OFF, //NOT USED
+    DSHOT_CMD_3D_MODE_ON, //NOT USED
+    DSHOT_CMD_SETTINGS_REQUEST, //NOT USED
+    DSHOT_CMD_SAVE_SETTINGS, //NOT USED
```

Comment:
I meant that they weren't being used for the blheli_s implementation right now and that there wasn't betaflight commands to send them yet.